PyratLabs/ansible-role-k3s
# feat: add kube-vip support

### Summary

Adds `kube-vip` support

### Issue type

- Feature

### Test instructions



Set `k3s_registration_address` to a VIP:

```yaml
k3s_registration_address: 192.168.1.139
```

Configure kube-vip:

```yaml
kubevip: true
kubevip_rbac: true
kubevip_interface: eno1
```

Make sure the tls-sans is updated:

```yaml
k3s_server:
  tls-san:
  - "{{ k3s_registration_address }}"
```

### Acceptance Criteria


Fixes: https://github.com/PyratLabs/ansible-role-k3s/issues/117
